src/extensions/WorkspaceView/SortingInstructions.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { FunctionComponent, useCallback, useState } from 'react';
```

Comment:
This is the Sorting Instructions panel that pops open (from the right) when the user clicks the "Spike Sorting" button.

![image](https://user-images.githubusercontent.com/3679296/107774819-83e8b300-6d0d-11eb-9dc4-ace7eab2b8e5.png)